pkg/lang/ir/parser/conda.go

```diff
@@ -24,6 +24,7 @@ type CondaEnv struct {
 	EnvName       string
 	CondaPackages []string
 	PipPackages   []string
+	Channels      []string
 }
 
 func ParseCondaEnvYaml(condaEnvFile string) (*CondaEnv, error) {
```

Comment:
I wonder if we should use the `env.yaml` directly instead of parsing it?

Previously we need to parse it because we don't have access to the `env.yaml` during building. But now we're able to mount the workspace during building.

Reply:
env.yaml contains the field `name`, which specifies the env name. But we hard coded the name to envd, thus there may be conflicts.

Reply:
env.yaml contains the field `name`, which specifies the env name. But we hard coded the name to envd, thus there may be conflicts.